scenarioo-client/test/protractorE2E/specs/show_step.js

```diff
@@ -146,10 +150,101 @@ useCase('Show step')
         scenario('Step without HTML source attached')
             .description('If the step data contains no html source data, the HTML tab should not be displayed at all')
             .it(function () {
-
                 stepPage.goToPage('/step/Donate/find_donate_page/startSearch.jsp/0/0');
                 stepPage.assertHtmlTabIsHidden();
                 step('A step with no HTML source attached');
             });
 
+
+        scenario('ScenarioPage with comparisons')
+            .description('Navigate back and forth through the scenario steps.')
+            .it(function () {
+                 usecasePage.goToPage('/scenario/Donate/find_donate_page?branch=wikipedia-docu-example&build=2014-03-19&comparison=To%20Projectstart&tab=usecases');
+                    stepPage.clickExpandAllPagesButton();
+                    stepPage.assertFirstChangedPageDiffIconHasValue();
+                    stepPage.assertFirstChangedStepDiffIconHasValue();
+                    stepPage.assertAddedStepDiffIconTextEqualsAdded();
+                    stepPage.assertRemovedStepDiffIconTextEqualsRemoved();
+                    stepPage.assertAddedPageDiffIconTextEqualsAdded();
+                    stepPage.assertRemovedPageDiffIconTextEqualsRemoved();
+                    step('Display one scenario');
+            });
+
+        scenario('Screenshot comparison options')
```

Comment:
@danielsuter My point was about the two test scenarios that intensively test the comparison view on a step (not the other one which was about the scenario overview). For me that is a use case on its own "show step" versus "compare step screenshots".